AlertCovidApp/src/screens/LogIn.js

```diff
@@ -32,32 +28,33 @@ const LogIn = ({navigation}) => {
         !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)
       ) {
         errors.email = 'Invalid Email';
-      }
-      else
-      {                
-        axios.post('https://ancient-citadel-41771.herokuapp.com/api/v1/login', {
-          "user":{
-          "email": values.email,
-          "password": values.password                        
-        }            
-        })
-        .then((response) => {         
-          var acceso=response.data.success        
-           console.log(response.data.user);
-           console.log(response.data.email);
-           console.log(response);
-          if (acceso === true){
-            navigation.navigate('HomeScreen')
-          }                 
-        }, (error) => {
-          Alert.alert("unregistered user!!")         
-        });
+      } else {
+        axios
+          .post( LogInConstants.UrlLogin, {
```

Comment:
I mean something like this:

```suggestion
          .post( `${LogInConstants.UrlLogin}logout`, {
```

Reply:
ok

Reply:
thanks